infrastructure/lib/workers/lambdas/delete_graph.py

```diff
@@ -13,31 +14,80 @@
 
 cluster_name = os.getenv("cluster_name")
 graph_table_name = os.getenv("graph_table_name")
+ec2 = boto3.client("ec2")
 
 
 def uninstall_release(helm_client, body):
     """
     Uninstalls a release from the Kubernetes Cluster
     """
-    release_Name = body["releaseName"]
+    release_name = body["releaseName"]
     expected_status=body["expectedStatus"]
 
     # Create a Graph object to track the deletion
-    graph = Graph(graph_table_name, release_Name)
+    graph = Graph(graph_table_name, release_name)
 
     if not graph.check_status(expected_status):
-        logger.warn("Graph %s had unexpected status. Abandoning delete", release_Name)
+        logger.warn("Graph %s had unexpected status. Abandoning delete", release_name)
         return
 
     graph.update_status("DELETION_IN_PROGRESS")
 
-    uninstalled = helm_client.uninstall_chart(release_Name)
+    uninstalled = helm_client.uninstall_chart(release_name)
     if uninstalled:
+        delete_volumes(release_name);
         graph.delete()
     else:
         graph.update_status("DELETION_FAILED")
 
-    
+
+def delete_volumes(release_name):
+    created_for_pvc_name_filter_values = [
```

Comment:
It might be cleaner to use the `kubectl` commands to do this: https://github.com/gchq/gaffer-docker/blob/develop/kubernetes/aws-eks-deployment.md#uninstall

Reply:
@d47853 made the changes as discussed.